app/javascript/react/components/SessionsListView/SessionsListView.style.tsx

```diff
@@ -1,18 +1,39 @@
 import styled from "styled-components";
 
-import { gray100 } from "../../assets/styles/colors";
+import { blue, gray100, gray400 } from "../../assets/styles/colors";
+import { SessionListTile } from "./SessionsListTile/SessionListTile.style";
 
 const SessionListViewStyle = styled.div`
   position: fixed;
   top: 10.5rem;
   bottom: 0;
   right: 1.25rem;
-  overflow-y: auto;
-  overflow-x: hidden;
   border-radius: 0.5rem;
   box-sizing: border-box;
   padding-right: 0.8rem;
-  z-index: 1;
+  z-index: 3;
```

Comment:
Do we need to change this z-index value? Now it's covering the thresholds:
<img width="774" alt="image" src="https://github.com/user-attachments/assets/84419ce7-87c9-4fe6-89ec-de0927822ef7">

Reply:
We need to change the zIndex. When the header wraps and has two rows, it covers the session list. This wasn't a problem until the popup came in. The popup will not show if there is another component with a higher index on top of it.